site/styles/_meta-detail.scss

```diff
@@ -297,7 +297,7 @@
   .meta-detail-country,
   .meta-detail-subtitle {
     padding-bottom: 20px;
-    h3, h4 {
+    h2, h3, h4 {
```

Comment:
Did you leave h3 and h4 here for backwards compatibility?

Reply:
Yes. I'll make sure I make the rest backwards compatable too.